### PR TITLE
Bugfix/user can get deleted

### PR DESCRIPTION
**Pull request by ahaurw01**
You can probably ignore the stuff in the session initializer since I'm about to remove the anonymous user business.

### CODE REVIEW

**Comment by ahaurw01** [comment]
stupid

```
Missing npm packages: 
Package: ember-paper
  * Specified: 0.0.21
  * Installed: 0.0.20
```

**Comment by ahaurw01** [comment]
Do you think we need to force something for it to reinstall packages somehow?

**Comment by SirZach** [comment]
i'm pretty sure the travis ci package we're using does an npm cache clean but... https://github.com/tonycoco/heroku-buildpack-ember-cli is the guy who needs investigating

**Comment by ahaurw01** [comment]
The `npm install` took all of 4.32s, which seems a little suspect to me.

**Comment by SirZach** [comment]
nah man, npm is just that good /s

**Comment by ahaurw01** [comment]
lol sorry

File: app/models/log-message.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import DS from 'ember-data';
+
+export default DS.Model.extend({
```
Comment (SirZach) [inline app/models/log-message.js]: what do you think about using a logging service like https://www.loggly.com/ or something similar? i'm just getting worried about that firebase bill

Comment (ahaurw01) [inline app/models/log-message.js]: Yeah, we could do that. Loggly kinda sucks, but would probably be more than enough for our purposes at the moment. Maybe i'll put together another PR for something like that? This was more/less just to debug really crazy stuff at the moment.

Comment (SirZach) [inline app/models/log-message.js]: :+1:

**Pull request merged**